recirq/benchmarks/rep_rate/rep_rate_calculator.py

```diff
@@ -142,14 +146,9 @@ def from_engine(cls,
             gate_set: gate set to use.  Defaults to square root of iswap
             processor_id: Processor to test on.
         """
-        gate_set = gate_set or cg.SQRT_ISWAP_GATESET
-        device = engine.get_processor(processor_id).get_device(
-            gate_sets=[gate_set])
-        sampler = engine.sampler(processor_id=processor_id, gate_set=gate_set)
-        gate = cirq.ISWAP**0.5
-        if gate_set == cg.SYC_GATESET:
-            gate = cg.SYC
-        return cls(device, sampler, gate)
+        device = engine.get_processor(processor_id).get_device()
+        sampler = engine.sampler(processor_id=processor_id)
+        return cls(device, sampler, cg.SYC)
```

Comment:
I think we still need to allow the user to specify `gate` since this changes the experiment. It's not just an argument to get an enginesampler

Reply:
@dstrain115 what do you think

Reply:
Yeah, I think we want to specify the two-qubit gate here.
For instance, syc and sqrt-iswap have different compilation paths into waveforms and have different durations, so they (theoretically) could have different rep rates.  Also, SYC may not be supported on all processors.

If anything, I think we should default to CZ going forward, not SYC.

Reply:
Updated